frontend/src/global_styles/content/_pagination.sass

```diff
@@ -79,7 +79,7 @@ $pagination--font-size: 0.8125rem
       min-width: 25px
       text-align: center
       padding: 3px 3px
-      background: #f8f8f8
+      background: var(--button--background-color)
```

Comment:
`--button--background-color` is also deprecated by now

Reply:
Hi! --button--background-color is one of our global variables, it's not deprecated, is it?

Reply:
Yes, your are right. My bad 🙈